replication/replication.proto

```diff
@@ -196,6 +211,9 @@ message GetVolumeReplicationInfoRequest {
   // This field SHALL be used by the CO in subsequent calls to refer to
   // this volume.
   string volume_id = 1;
+  // If specified, this field will contain volume or volume group id
+  // for replication. This field is OPTIONAL.
```

Comment:
This cannot be optional, right? This need to be set to `VolumeSource` or `VolumeGroupSource` isn't it

Reply:
replication_source field is optional if we are preserving volume_id field for backwards compatibility. If we decide to deprecate the volume_id and use the replication_source as the main object to identify it's a volume replication or the group replication - it must be changed to mandatory.